### ISSUE TITLE
Error exporting  to SVG - <path> is missing an SPACE

### ISSUE THREAD

**Issue by Igcorreia** [fabricjs/fabric.js#5503]
Hi, when exporting to SVG we detected an error on the PATH element.

![image](https://user-images.githubusercontent.com/1076409/51603843-df675e80-1f02-11e9-8bc6-e9f27fe2c7e0.png)

So we dig into the code and we found this:

```
<g transform="matrix(-1 0 0 -1 79.5 -15.5)" id="2_1_0"  >
<path d="M 29.5 0 A 29.5 29.5 0 0 1 -29.5 0"style="stroke: rgb(243,242,242); stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-miterlimit: 4; fill: rgb(243,242,242); fill-rule: nonzero; opacity: 1;"   />
</g>
```

Please noticed that the d= property is connected to the style= property, it is just missing an space.

So to fix we added the space in this line:
https://github.com/fabricjs/fabric.js/blob/f1459f3a1d2ee7eee8e7f4642e073d1abe72d487/src/shapes/circle.class.js#L109

changed from:
` '"', 'COMMON_PARTS', ' />\n'`

to:
` '" ', 'COMMON_PARTS', ' />\n'`

noticed the SPACE after the first ".

Result after:

![image](https://user-images.githubusercontent.com/1076409/51604305-4f2a1900-1f04-11e9-84aa-2c8e3f764af6.png)

**Comment by asturur** [comment]
hey thanks for spotting this.
Could you please open a PR to fix it?

**Comment by asturur** [comment]
closed with #5544
